website/docs/developer/research/research-orca-delete-functionality.md

```diff
@@ -0,0 +1,122 @@
+can be used from the boto3 library. An example of the usage can be seen below.
+Some notable items if this library is used includes the following.
+- It can be used to delete up to 1000 objects from a bucket using a single API call.
+- When performing this action on an MFA Delete enabled bucket that attempts to delete any versioned objects, an MFA token must be included orelse it will fail.
+- If the object specified in the request is not found, Amazon S3 returns the result as deleted.
+- Each DeleteObjects API incurs a little cost.
+
+#### Example Using delete_objects
+
+```python
+import boto3
+
+s3_client = boto3.client('s3')
+bucket = "riz-orca"
+response = s3_client.delete_objects(
+    Bucket=bucket,
+    Delete={
+        'Objects': [
+            {
+                'Key': 'MOD09A1.006/MOD09A1.A2019025.h07v03.006.2019037205734.hdf',
+            }
+        ]
+    }
+)
+print(response)
+
+# Results
+{
+   "ResponseMetadata":{
+      "RequestId":"KJCZAPM0RWMJNNC7",
+      "HostId":"z2fRlgMxT10ApzcyDOLp3REn3PjM71OSgdH0oPpe4v2pHchmSTPCfEjBjTx00PyTL55wgmQ7gF4=",
+      "HTTPStatusCode":200,
+      "HTTPHeaders":{
+         "x-amz-id-2":"z2fRlgMxT10ApzcyDOLp3REn3PjM71OSgdH0oPpe4v2pHchmSTPCfEjBjTx00PyTL55wgmQ7gF4=",
+         "x-amz-request-id":"KJCZAPM0RWMJNNC7",
+         "date":"Tue, 08 Mar 2022 05:19:07 GMT",
+         "content-type":"application/xml",
+         "transfer-encoding":"chunked",
+         "server":"AmazonS3",
+         "connection":"close"
+      },
+      "RetryAttempts":0
+   },
+   "Deleted":[
+      {
+         "Key":"MOD09A1.006/MOD09A1.A2019025.h07v03.006.2019037205734.hdf"
+      }
+   ]
+}
+```
+### Logging
+
+Additional logging should be collected during the deletion process to keep records for safety and security purposes. Some logging to capture might include who deleted the data, when how many granules were deleted over a particular timespan, from which collections was it deleted, and what workflow was used to delete them.
+
+Cloudwatch logs and cloudtrail can be used to collect logging information. In addition, S3 server access logging can be used at the bucket level which would provide detailed records for the requests that are made to a bucket. The log information can be useful in security and access audits. AWS will collect access log records, add the records in log files, and then upload log files to the target bucket as log objects. There is no extra charge for using the service except the cost for storing the log files in S3.
+
+## Future directions
+
+Few cards have been created to perform the ORCA delete functionality work based on initial research. This should be revisited during PI planning/team meetings to confirm the use cases/requirements before starting the work.
+Some of the cards created to finish the task include:
+- https://bugs.earthdata.nasa.gov/browse/ORCA-394- Create Lambda function for performing soft deletes from ORCA
```

Comment:
Before you create Lambdas and things you will likely want to talk over the design, especially with authentication and business rules.